src/api/index.ts

```diff
@@ -19,6 +20,7 @@ export interface SchemaRegistryAPIClientArgs {
   auth?: Authorization
   clientId?: string
   retry?: Partial<RetryMiddlewareOptions>
+  forSchemaOptions?: Partial<ForSchemaOptions>
```

Comment:
This does not belong with the APIClient configuration

Reply:
Moved to separate object in latest push.